documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    str1 + str2                 |    concatenate(str1, str2)                    |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    list(str)                   |    as(<list>, str)                            |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    list(str)                   |    as(<string>, chars)                        |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    copy(str)                   |    shallow-copy(str)                          |
+|                                |    copy-sequence(str)                         |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    # Strings are immutable     |    fill!(str, char)                           |
++--------------------------------+-----------------------------------------------+
+
+Vectors
+^^^^^^^
++--------------------------------+-----------------------------------------------+
+| Python                         | Dylan                                         |
++================================+===============================================+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    isinstance(obj, list)       |    instance?(obj, <vector>)                   |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    [fill] * k                  |    make(<vector>, size: k, fill: fill)        |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    [obj ...]                   |    vector(obj, ...);                          |
```

Comment:
The Python side needs a comma.  The Dylan side doesn't really need the semicolon.